src/Components/Facility/ConsultationForm.tsx

```diff
@@ -200,19 +256,26 @@ export const ConsultationForm = (props: any) => {
         discharge_date: state.form.discharge_date,
         patient: patientId,
         facility: facilityId,
-        referred_to: state.form.suggestion === 'R' ? state.form.referred_to : undefined,
+        referred_to:
```

Comment:
Codacy found an issue: [Identifier 'referred_to' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5811785)